src/NewUser.js

```diff
@@ -0,0 +1,88 @@
+import axios from 'axios';
+import { Form, Input, Button, Drawer } from 'antd';
+
+
+const NewUser = (props) => {
+
+    const [form] = Form.useForm();
+
+    const handleAddUser = values => {
+        const newUser = {...values};
```

Comment:
There is no need to have a shallow clone of values you use it right away.
If you want to make it more descriptive you can just name values to newUser instead.